CONTRIBUTING.md

```diff
@@ -8,152 +8,104 @@
+## Security Bugs
+If you believe you have found a security bug, [create a GitHub issue](https://github.com/vmware/versatile-data-kit/issues) and reach out to us in the [Contacts Us](#contact-us) section.
 
-* Use forks to only contribute changes on examples and documentation.
-   * Currently we accept contribution from forks only for examples and documentation changes until PR 854 is fixed. Until then please request write priveleges and create a branch in the main repo as described below.
-* Create a topic branch from where you want to base your work - name your branch according to our naming convention - person/<github-username>/<feature-name>
-* Make commits of logical units
-* Make sure your commit messages are with the proper format, quality and descriptiveness (see below)
-* Push your changes to the topic branch in your fork
-* Create a pull request containing that commit
+# Propose a Change
+Before suggesting a feature or change, consider the impact of this change. Does it only serve your needs, or does it serve the broader community's needs?
 
-We follow the GitHub workflow and you can find more details on the [GitHub flow documentation](https://docs.github.com/en/get-started/quickstart/github-flow).
+For substantial features or changes, submitting your design is valuable. Creating a PR in Github with your proposal as markdown in the [specs](https://github.com/vmware/versatile-data-kit/blob/main/specs) directory is recommended. Reviews, feedback, and approvals are documented in the PR.
 
-## Structure of the project
+Reach out to the community through Slack or e-mail in the [Contact Us](#contact-us) section to discuss your idea. We are happy to help.
 
-* projects - the interesting part, aka the source code - there is a separate folder for each component;
-  * control-service - Java spring-based API for managing the lifecycle of data jobs: job definitions, job deployments, and executions in Kubernetes;
-  * vdk-core - Python-based SDK containing a data library for developing and running data jobs. Includes a powerful plugin framework;
-  * vdk-heartbeat - tool for verifying the deployed SDK and Control Service are functional and working correctly;
-  * vdk-control-cli - User friendly CLI interface over Control Service operations including login/logout;
-  * vdk-plugins - Set of plugins that we maintain and provide for different use-cases like lineage, database support, etc.;
-  * frontend - Angular web UI for creating, deploying and managing jobs.
-* support - helper scripts used by developers of the project during their workday;
-* cicd - build and ci cd related scripts common across all projects. Each project also has its own cicd folder;
-* examples - list of example use-cases. Each example directory has its README with detailed explanations;
-* specs - specs for feature proposals and architecture documents;
-* events - details about the conferences/workshops where VDK was presented.
```

Comment:
Do you think this may be duplicate info for removal from the Contributing guide?

Reply:
This section will be removed.

Reply:
Well the section explains the main project structure, and it may be of value to have this information somewhere. In case deleted here, I suppose it already exists somewhere else already?

Reply:
Ah - yes I understand now. Before I merge, we will need to decide where these sections need to go:
- Structure of the project
- How to build, debug
- How to make a new VDK release
- CI

Reply:
@ivakoleva I've created a 3 step list that specifically mentions VEP.

Reply:
May I propose creating a ticket, linked to this PR changes, so we may create VEPs/READMEs/tutorials on the topics enumerated? Structure of the project may be a specification, for examples see the PEP index spec contents https://peps.python.org/pep-0000/

May the rest of the reviewers share their thoughts on this subject?